catalog/imaging/imaging-plugin-nitf/src/main/java/org/codice/alliance/plugin/nitf/NitfPreStoragePlugin.java

```diff
@@ -471,4 +471,8 @@ public void setCreateOverview(boolean createOverview) {
     public void setStoreOriginalImage(boolean storeOriginalImage) {
         this.storeOriginalImage = storeOriginalImage;
     }
+
+    protected NitfRenderer getNitfRenderer() {
```

Comment:
package-private should do the trick. I'm not sure that injecting it would be thread-safe so for now I think this is the safest solution.